### ISSUE TITLE
Inform Charles

### ISSUE THREAD

**Issue by TanNgocDo** [DinosaurGroup/BuddiesApp#1]
On behalf of Khung Long Group, I have submit the 1st milestone. Could you please check it for me /cc @chug2k ?

**Comment by chug2k** [comment]
please allow sang@coderschool.vn to view the Google Doc

**Comment by chug2k** [comment]
I would like to see the wireframes, but I have some early feedback:

I think the idea is too broad. What is your **story**? Why would people want to find a Buddy? Are they lonely, do they need help, are they hungry, do they want romance, do they want to learn a new language, do they want to meet in person, do they want to chat online, do they want a hobby partner? 

I personally like having a few target customers in mind. Imagine you are telling people to download your app. What are you telling them? "Download my app because .... <????>"

Right now, I feel like your app is a little bit like Tinder. Why is your app better than Tinder? 

Also, the biggest problem with any application like this is having enough people to start. The more options you give people to filter (nationality, hobbies, etc) the more people you need to download the app to make it useful. (This is the same problem for dating apps.) How will you solve this problem? (This is called the *critical mass* problem.)

**Comment by TanNgocDo** [comment]
Hello, I misunderstood about user stories. I will update

**Comment by chug2k** [comment]
No, no, your user stories are correct. Sorry this English is confusing!

User stores = tasks 

*Story* = just the feeling you want the user to have when they use your app.

**Comment by TanNgocDo** [comment]
hey, the readme has just updated and sharing acces google drive. Could you
please check ?

**Comment by chug2k** [comment]
yes, thank you!

**Comment by chug2k** [comment]
I still have the questions from my earlier comment. :question:

**Comment by nttan251290** [comment]
Hi Charles, I will update the layout tonight . The design now is not enough :( .

**Comment by chug2k** [comment]
You will be showing this to some people in class today...

**Comment by TanNgocDo** [comment]
updated the wireframe
